### PR TITLE
tDatD: Kill Switch

### PR DESCRIPTION
**Pull request by msbeck**
I looked at J:PE for an effect when an agenda is scored. Credit Crash for "when accessed", and Kuwinda for a trace resulting in brain damage.

screen shot 2018-04-05 at 11 56 11 am
Corp Scores, wins trace
screen shot 2018-04-05 at 11 57 53 am
Runner steals, wins trace.

### CODE REVIEW

File: src/clj/game/cards/operations.clj [hunk 1]
```diff
@@ -678,6 +678,13 @@
    "IPO"
    {:msg "gain 13 [Credits]" :effect (effect (gain :credit 13))}
 
+   "Kill Switch"
+   (let [trace-for-brain-damage {:trace {:base 3 :msg "give the Runner 1 brain damage"
+                                        :delayed-completion true
+                                        :effect (effect (damage :runner eid :brain 1 {:card card}))}}]
+   {:events {:pre-access-card (assoc trace-for-brain-damage :req (req (is-type? target "Agenda")))
+             :agenda-scored trace-for-brain-damage}})
```
Comment (NoahTheDuke) [inline src/clj/game/cards/operations.clj]: This and the previous line should be indented two spaces.

File: src/clj/game/cards/operations.clj [hunk 2]
```diff
@@ -678,6 +678,13 @@
    "IPO"
    {:msg "gain 13 [Credits]" :effect (effect (gain :credit 13))}
 
+   "Kill Switch"
+   (let [trace-for-brain-damage {:trace {:base 3 :msg "give the Runner 1 brain damage"
+                                        :delayed-completion true
+                                        :effect (effect (damage :runner eid :brain 1 {:card card}))}}]
```
Comment (NoahTheDuke) [inline src/clj/game/cards/operations.clj]: This and the previous line should be indented one space to line up with `:base`.

File: src/clj/game/cards/operations.clj (hunk 1 above)
Comment (msbeck) [reply src/clj/game/cards/operations.clj]: done

File: src/clj/game/cards/operations.clj (hunk 2 above)
Comment (msbeck) [reply src/clj/game/cards/operations.clj]: done

**Comment by nealterrell** [comment]
Nice job!

**Pull request merged**